test_opensearchpy/test_client/test_point_in_time.py

```diff
@@ -0,0 +1,46 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+#
```

Comment:
You don't need an ES license grant for new code, unless it's copied from ES, remove this part.

Reply:
Removed thanks!